guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<blockquote><em>The </em><code><em>number_of_predictions</em></code><em> parameter controls the number of predictions to append to </em><strong><em>the</em></strong><em> existing </em><strong><em>the</em></strong><em> sequence,</em></blockquote><blockquote>remove duplicate <strong>the </strong>from the above line</blockquote><blockquote><em>The values of all rasters for a given pixel are used to predict the value of that </em><strong><em>particular</em></strong><em> pixel in the output raster.</em></blockquote><blockquote>remove the word<em> </em><strong>particular<em> </em></strong> in the above line</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>